source/2017-04-13-how-we-got-here.html.markdown.erb

```diff
@@ -0,0 +1,145 @@
+---
+title: Bandwith, coverage, monitoring and other types of Voodoo
+date: 2017-04-13 13:37 UTC
+author: stb
+tags:
+---
+
+As a sysadmin at Panter you’re also responsible for the infrastructure of [Impact
+Hub Zürich](https://zurich.impacthub.ch) a vibrant coworking space in the heart of Zürich. On 3 floors and a
+total of roughly 2000 square meters, we’re making sure that all the members fit into our WiFi
+network. As always in such a crowded place there’s not much margin for errors.
+And because our frequencies are almost as crowded with rogue WiFis like the death
+star’s having holes for thermonuclear missiles, this can be quite challenging and
+sometimes quite frustrating.
+
+[![Youtube:Steve Jobs 500 WiFi Basestations](jobs.jpg)]
+(https://youtu.be/1-oIL9cLHDc?t=19 "Steve Jobs WiFi")
+
+In this blog post we'd like to give you a little bit of insight into our configuration
+and how we're troubleshooting our network. 
+Our WiFi runs exclusively on [Mikrotik](https://mikrotik.com/) devices because this gives us a lot of flexibility. Their devices come at a fair
+price and we're not forced to pay fees to use their products.
+When we first came to the new Colab we had nothing but an empty rack, an old DIY router and
```

Comment:
"Colab" might not mean anything to the reader, maybe write "new coworking space"?